src/Elastic.Apm/Logging/AbstractLogger.cs

```diff
@@ -12,7 +12,7 @@ public abstract class AbstractLogger
 		/// they need to write it (e.g. into a file, to the console)
 		/// </summary>
 		/// <param name="logline">This line that must be logged - it already contains the prefix and the loglevel</param>
-		protected abstract void PrintLogline(string logline);
+		protected abstract void PrintLogLine(string logLine);
```

Comment:
Good catch! would still like to see https://github.com/elastic/apm-agent-dotnet/issues/61 land in the first alpha but understand the time pressure with v1 vs v2 of the intake API.

Reply:
I understand. #61 can really break the API, still I'd not consider it as high prio compared to other things we have now - plus replacing our logger is something that I think not many users would do. (Of course I'm not holding back anyone to propose a nicer API as early as possible) :)